i18n/md/collections/2023/en/categories/build.md

```diff
@@ -0,0 +1,21 @@
+---
+id: build
+language: en
+---
+
+{bun} is also a [bundler](https://bun.sh/docs/bundler) so we included it this year in the "Building Tools" category.
+
+At two {vite} is still the most popular versatile bundler, powering meta frameworks such as Astro, Nuxt, Remix, SolidStart, SvelteKit...
+
+The [version 5](https://vitejs.dev/blog/announcing-vite5) released in November, is built on top of {rollup} 4, bringing build performance improvement.
+
+Evan You [announced](https://twitter.com/youyuxi/status/1709943106215530867) he was working on Rolldown a port of Rollup in Rust that will be integrated to Vite. So the future of Vite is bright!
+
+At three, the rise of {biome} is one of the stories of the year. It's related to the [fail of Rome](https://bytes.dev/archives/175), an ambitious project that aimed to unify the tooling in JS projects (compile, lint, format, bundle, test). The company behind the project failed and the project was stopped this year.
```

Comment:
Not sure if you meant to say "fall of Rome" instead of "fail of Rome", but I would suggest using "fall of Rome"

Reply:
Yes, that's a typo, I wanted a dramatic effect, that relates to the fall of the Roman Empire 😄